### DISCUSSION TITLE
Simülasyon Nedir | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#353] [Site]
# Simülasyon Nedir | SonsuzUs Github

Simülasyonlar

https://sonsuzus.github.io/posts/simulasyon-nedir/

**Comment by sonsuzus** [comment]
Olasılık soruları çözülürken olayın uzayda kaç defa gerçekleştiğini de bulmak için simüle ediliyor.
